packages/jest-preset-mc-app/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @commercetools-frontend/jest-preset-mc-app
 
+## 16.12.1
```

Comment:
Huh. This can be removed. Sorry. This file is "deprecated" automation does this these days :)

Reply:
This is also not mine, bot did it for me :)

Reply:
Ah my bad.